.vscode/settings.json

```diff
@@ -31,7 +31,7 @@
     "editor.defaultFormatter": "redhat.vscode-yaml"
   },
   "gopls": {
-    "build.buildFlags": ["-tags=or_test,or_dev,or_e2e,or_int"]
+    "build.buildFlags": ["-tags=or_test,or_dev,or_int"]
```

Comment:
What is this doing? It's not in the PR description. Add it?

Reply:
Done